packages/components/src/ErrorState/ErrorState.tsx

```diff
@@ -1,12 +1,25 @@
 import React from 'react';
-import { Stack, StackItem } from '@patternfly/react-core';
 import { ExclamationCircleIcon } from '@patternfly/react-icons/';
 import DefaultErrorMessage from './DefaultErrorMessage';
-import propTypes from 'prop-types';
-import { Title, EmptyState, EmptyStateVariant, EmptyStateIcon, EmptyStateBody, Button } from '@patternfly/react-core';
+import {
+  Title,
+  EmptyState,
+  EmptyStateVariant,
+  EmptyStateIcon,
+  EmptyStateBody,
+  Button,
+  EmptyStateProps,
+  Stack,
+  StackItem,
+} from '@patternfly/react-core';
 import './error-state.scss';
 
-const ErrorState = ({ errorTitle, errorDescription, ...props }) => {
+export interface ErrorStateProps extends EmptyStateProps {
+  errorTitle: string,
+  errorDescription?: string;
```

Comment:
This has come to my attention that this prop type was incorrect. Can you change it to `React.ReactNode`? That will allow all react nodes to be used as a description instead of just strings.